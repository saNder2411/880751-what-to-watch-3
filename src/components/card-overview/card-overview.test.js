import React from 'react';
import renderer from 'react-test-renderer';
import CardOverview from './card-overview.jsx';

const data = {
  description: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.`,
  rating: 9,
  scoresCount: 100,
  director: `Steven Spielberg`,
  starring: [`Judi Dench`, `Robert De Niro`, `Leonardo DiCaprio`],
};

it(`Should CardOverview render correctly`, () => {
  const markup = renderer
    .create(<CardOverview {...data} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
