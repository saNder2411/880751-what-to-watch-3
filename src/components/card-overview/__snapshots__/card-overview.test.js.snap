// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CardOverview render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        100
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Steven Spielberg
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Judi Dench, Robert De Niro, Leonardo DiCaprio
      </strong>
    </p>
  </div>,
]
`;
